src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/BlobProcessorTest.java

```diff
@@ -77,9 +76,13 @@ public void should_process_zipfile_after_upload_and_set_status() throws Exceptio
         assertThat(envelopeListResponse.envelopes.size()).isGreaterThanOrEqualTo(1);
 
         assertThat(envelopeListResponse.envelopes)
-            .extracting("zipFileName", "status")
-            .containsOnlyOnce(tuple(destZipFilename, Status.NOTIFICATION_SENT));
+            .extracting("zipFileName")
+            .containsOnlyOnce(destZipFilename);
 
+        assertThat(envelopeListResponse.envelopes
+            .stream().filter(e -> destZipFilename.equals(e.getZipFileName()))
+            .map(e -> e.getStatus()))
+            .containsOnlyOnce(Status.NOTIFICATION_SENT);
```

Comment:
`containsExactly` would make more sense imho (`['NOTIFICATION_SENT', 'A, 'B, 'C']` would pass this assertion)